lib/t/feeds.ex

```diff
@@ -145,6 +147,20 @@ defmodule T.Feeds do
     end
   end
 
+  defp with_distance(query, location) do
+    if location do
+      select_merge(query, [p], %{
+        distance:
+          fragment(
+            "ST_DistanceSphere(?::geometry, location::geometry)::integer / 1000",
```

Comment:
::integer + integer division ensures non-accuracy of distance (impossible to triangulate the exact location)